homeassistant/components/anova/sensor.py

```diff
@@ -0,0 +1,110 @@
+        icon="mdi:clock-outline",
+        name="Cook Time",
+    ),
+    SensorEntityDescription(key=AnovaPrecisionCookerSensor.STATE, name="State"),
+    SensorEntityDescription(key=AnovaPrecisionCookerSensor.MODE, name="Mode"),
+    SensorEntityDescription(
+        key=AnovaPrecisionCookerSensor.TARGET_TEMPERATURE,
+        native_unit_of_measurement=UnitOfTemperature.CELSIUS,
+        device_class=SensorDeviceClass.TEMPERATURE,
+        state_class=SensorStateClass.MEASUREMENT,
+        icon="mdi:thermometer",
+        name="Target Temperature",
+    ),
+    SensorEntityDescription(
+        key=AnovaPrecisionCookerSensor.COOK_TIME_REMAINING,
+        native_unit_of_measurement=UnitOfTime.SECONDS,
+        icon="mdi:clock-outline",
+        name="Cook Time Remaining",
+    ),
+    SensorEntityDescription(
+        key=AnovaPrecisionCookerSensor.HEATER_TEMPERATURE,
+        native_unit_of_measurement=UnitOfTemperature.CELSIUS,
+        device_class=SensorDeviceClass.TEMPERATURE,
+        state_class=SensorStateClass.MEASUREMENT,
+        icon="mdi:thermometer",
+        name="Heater Temperature",
+    ),
+    SensorEntityDescription(
+        key=AnovaPrecisionCookerSensor.TRIAC_TEMPERATURE,
+        native_unit_of_measurement=UnitOfTemperature.CELSIUS,
+        device_class=SensorDeviceClass.TEMPERATURE,
+        state_class=SensorStateClass.MEASUREMENT,
+        icon="mdi:thermometer",
+        name="Triac Temperature",
+    ),
+    SensorEntityDescription(
+        key=AnovaPrecisionCookerSensor.WATER_TEMPERATURE,
+        native_unit_of_measurement=UnitOfTemperature.CELSIUS,
+        device_class=SensorDeviceClass.TEMPERATURE,
+        state_class=SensorStateClass.MEASUREMENT,
+        icon="mdi:thermometer",
+        name="Water Temperature",
+    ),
+]
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    entry: config_entries.ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up Anova device."""
+    anova_data: AnovaData = hass.data[DOMAIN][entry.entry_id]
+    sensors: list[AnovaSensor] = []
+    for coordinator in anova_data.coordinators:
+        sensors.extend(
+            [
+                AnovaSensor(coordinator, description)
+                for description in SENSOR_DESCRIPTIONS
+            ]
+        )
+    async_add_entities(sensors)
+
+
+class AnovaSensor(AnovaEntity, SensorEntity):
+    """A sensor using anova coordinator."""
+
+    def __init__(
+        self, coordinator: AnovaCoordinator, description: SensorEntityDescription
+    ) -> None:
+        """Set up an Anova Sensor Entity."""
+        super().__init__(coordinator)
+        self.entity_description = description
+        self._sensor_update_key = description.key
+        self._sensor_data = None
+        self._attr_unique_id = f"{coordinator._device_unique_id}_{description.key}"
+        self._attr_device_info = coordinator.device_info
+        self._attr_has_entity_name = True
```

Comment:
```suggestion
```

This is already set in the parent class
https://github.com/home-assistant/core/pull/86254/files#diff-30557a37c4c85ca7f9a59d5003ce3c09ec6c3ebad277e2f7dc3fd8aa9b7f1072R17

Reply:
Good catch - added entity class but didn't remove it here.